### ISSUE TITLE
Set new tolerance level on the fly?

### ISSUE THREAD

**Issue by vpatton** [Huddle/PhantomCSS#132]
If it's not available now, it would be nice to be able to set a new tolerance level at different times in the test, pics do not require as strict a tolerance level, as say comparing fonts in a paragraph.

**Comment by jamescryer** [comment]
No, that's not possible right now. But I'd accept a PR to add the feature.
